tests/README.md

```diff
@@ -0,0 +1,202 @@
+
+
+2. Install [JUST](https://github.com/casey/just) command runner
+
+```shell
+brew install just
+```
+
+After installing just you will be able to execute commands defined in **JUSTFILE** located in the project root folder.
+
+2. Install [kind](https://kind.sigs.k8s.io/docs/user/quick-start)
+
+```shell
+brew install kind
+```
+After installing kind you will be able to create and run local Kubernetes clusters using Docker container.
+
+3. Install [Helm](https://helm.sh/docs/intro/install/)
+
+```
+brew install helm
+```
+Helm is a package manager for Kubernetes.
+
+4. Install [Allure](https://docs.qameta.io/allure/#_installing_a_commandline) commandline tool
+
+```shell
+brew install allure
+```
+After install allure commandline you will be able to display raw test results in pretty html format. 
+
+## System Under Test (SUT) Setup
+
+Before performing steps below verify that **just** tool is installed and your root folder in commandline is cloudbeat.
+1. Create Kubernetes cluster
+```shell
+just create-kind-cluster
+```
+This command will create new kubernetes cluster named **kind-mono**.
+In order to verify that cluster installed you can execute:
+```shell
+kind get clusters
+```
+
+3. Build cloudbeat and upload docker image to kind
+```shell
+just build-cloudbeat
+just load-cloudbeat-image
+```
+
+4. Install elasticsearch and start cloudbeat
+```shell
+just deploy-tests-helm
+```
+This command will install elasticsearch 7.17.7 one node instance in kubernetes cluster and also will start cloudbeat
+
+
+## Developing Tests
+
+### Built With
+
+Automated test framework is built on top of the following:
+- [pytest](https://docs.pytest.org/en/7.1.x/) - mature full-featured Python testing framework.
+- [poetry](https://python-poetry.org/docs/) - tool for dependency **management** and **packaging** in Python.
+- [allure](https://docs.qameta.io/allure/#_pytest) - test report tool and framework that generates html reports, and allow everyone to participate in the development process to extract maximum of useful information from tests.
+
+### Prerequisites
+
+Install [poetry](https://python-poetry.org/docs/#installation) python package manager
+```shell
+curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python -
+```
+**Note**: Installing poetry process will also initiate python installation if not installed yet. 
+
+### Setting up Dev
+
+In the github, fork [cloudbeat repo](https://github.com/elastic/cloudbeat). After forking the repo, clone and install dependencies.
+
+```shell
+git clone https://github.com/<yourname>/cloudbeat.git
+cd cloudbeat/tests
+poetry install
+```
+### Configuring IDE
+
+Tests development may be done using any IDE like [visual studio code](https://code.visualstudio.com/) or [pycharm](https://www.jetbrains.com/pycharm/promo/?source=google&medium=cpc&campaign=14124132918&gclid=CjwKCAjw9LSSBhBsEiwAKtf0n1GGB55o_rxfYFVmBzRYUqbz_g5YhZdP8Th1iKXFsaRl43rEz_h7ehoCl2wQAvD_BwE).
```

Comment:
nit remove parameters from link. looks too suspicious 
https://www.jetbrains.com/pycharm/

Reply:
The link will be updated in the next PR.